documentation/src/main/wiki/Dictionary.md

```diff
@@ -4,10 +4,10 @@
 a *JMS (Java Message Service)* Server which is a basic communication channel between AET System components.
 
 *AET*  
-an acronym for **A**utomatic **E**xploratory **T**esting, a testing tool developed by Cognifide.
+an open source testing tool developed by Cognifide.
 
 *AET Core*  
-a set of system modules that are crucial to whole system work. The AET system will not work properly without all core modules configured and running properly.
+a set of system modules that are crucial to the whole system to work. The AET system will not work properly without all core modules configured and running properly.
```

Comment:
> crucial to the whole system to work

sounds a bit strange. 

How about

> crucial for the whole system to work

or

> crucial to the operation of the whole system

I'm just following a gut feeling here, struggling to find some clear guidelines by a credible source.